CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 ## [Unreleased]
 
+
+## [2024-06-18]
+
 ### Added
 
 * ParkAPI: [Multiple new converters at ParkAPI Sources 0.5.0](https://github.com/ParkenDD/parkapi-sources-v3/blob/main/CHANGELOG.md#050)
```

Comment:
It's Sources 0.5.2 now, no?

Reply:
Yea, but the converters came with 0.5.0. It's 0.5.2 because of two bugfixes at one of the converters.